src/pages/palanbyulChoi/Login/Login.js

```diff
@@ -1,8 +1,105 @@
 import React, { Component } from 'react';
+import './Login.scss';
 
 class LoginPalanbyul extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      disabled: true,
+    };
+  }
+
+  activeLogin = () => {
+    this.state.id.indexOf('@') > 0 && this.state.pw.length >= 5
+      ? this.setState({ disabled: false })
+      : this.setState({ disabled: true });
+  };
+
+  handleInput = e => {
+    const { value, id } = e.target;
+    this.setState(
+      {
+        [id]: value,
+      },
+      this.activeLogin
+    );
+  };
+  goToMain = () => {
+    if (!this.state.disabled) {
+      this.props.history.push('/main-palanbyul');
+    }
+  };
+  con = () => {
+    fetch('http://10.58.0.171:8000/users/sign-in', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+        name: '테드하',
+        phone_number: '010-2222-3333',
+        date_of_birth: '1998-12-15',
+      }),
+    })
+      .then(response => response.json())
+      .then(response => {
+        console.log(response);
+        if (response.message === 'SUCCESS') {
+          alert('성공');
+        } else if (response.message === 'INVALID_USER') {
+          alert('이메일과 비밀번호를 다시 확인해주세요');
+        }
+      });
+  };
+
   render() {
-    return <div>Palanbyul</div>;
+    return (
+      <div className="LoginPalanbyul">
```

Comment:
- 컴포넌트 이름만 대문자 카멜케이스로 작성하시고, 이번 보통의 이름은 소문자 카멜 케이스로 작성해주세요!
```suggestion
      <div className="loginPalanbyul">
```

Reply:
넵 !!